Src/BootCamp.Chapter/Person.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace BootCamp.Chapter
+{
+    internal class Person
+    {
+        private string _name;
+        private decimal _amount;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_amount' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)